src/supernode/baseclientproxy.cpp

```diff
@@ -114,48 +117,50 @@ bool supernode::BaseClientProxy::CreateAccount(const supernode::rpc_command::CRE
     std::string seed;
     wal->get_seed(seed);
     out.Seed = seed;
-    out.Result = STATUS_OK;
     return true;
 }
 
-bool supernode::BaseClientProxy::GetSeed(const supernode::rpc_command::GET_SEED::request &in, supernode::rpc_command::GET_SEED::response &out)
+bool supernode::BaseClientProxy::GetSeed(const supernode::rpc_command::GET_SEED::request &in, supernode::rpc_command::GET_SEED::response &out, epee::json_rpc::error &er)
 {
-	LOG_PRINT_L0("BaseClientProxy::GetSeed" << in.Account);
+    LOG_PRINT_L0("BaseClientProxy::GetSeed " << in.Account);
     std::unique_ptr<tools::GraftWallet> wal = initWallet(base64_decode(in.Account), in.Password);
     if (!wal)
     {
-        out.Result = ERROR_OPEN_WALLET_FAILED;
+        er.code = ERROR_OPEN_WALLET_FAILED;
+        er.message = ERROR_MESSAGE(MESSAGE_OPEN_WALLET_FAILED);
         return false;
     }
     wal->set_seed_language(in.Language);
     std::string seed;
     wal->get_seed(seed);
     out.Seed = seed;
-    out.Result = STATUS_OK;
     return true;
 }
 
-bool supernode::BaseClientProxy::RestoreAccount(const supernode::rpc_command::RESTORE_ACCOUNT::request &in, supernode::rpc_command::RESTORE_ACCOUNT::response &out)
+bool supernode::BaseClientProxy::RestoreAccount(const supernode::rpc_command::RESTORE_ACCOUNT::request &in, supernode::rpc_command::RESTORE_ACCOUNT::response &out, epee::json_rpc::error &er)
 {
-	LOG_PRINT_L0("BaseClientProxy::RestoreAccount" << in.Seed);
+    LOG_PRINT_L0("BaseClientProxy::RestoreAccount " << in.Seed);
```

Comment:
Might be a security issue (seed in the logs)

Reply:
Removed seed